modules/Quiz/resources/css/quiz.css

```diff
@@ -231,7 +233,7 @@
     cursor: default!important;
 }
 .display-question:focus{
-    outline: none;
+    /*outline: none;*/
```

Comment:
@MaksymKorniiov  
If we decide that this is not relevant anymore - let's delete it.

Reply:
@eitanavgil fixed it in new commit